active/TestInsecureHTTPVerbs.py

```diff
@@ -18,7 +18,7 @@
 
 """
 
-insecureverbs=["HEAD", "OPTIONS", "PUT", "DELETE", "TRACE", "CONNECT", "DEBUG", "MOVE", "SEARCH", "PATCH", "MKCOL", "COPY", "LOCK", "UNLOCK", "ARBIT", "XXXX", "12AB"]
+insecureverbs=["HEAD", "TRACE", "OPTIONS", "PUT", "DELETE", "CONNECT", "DEBUG", "MOVE", "SEARCH", "PATCH", "MKCOL", "COPY", "LOCK", "UNLOCK", "ARBIT", "XXXX", "12AB"]
```

Comment:
Sure, do you want me to add the comment right inside the code or in this thread ?